pkg/aws/idps.go

```diff
@@ -23,21 +23,29 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/openshift/rosa/pkg/aws/tags"
 )
 
 const (
 	OIDCClientIDOpenShift = "openshift"
 	OIDCClientIDSTSAWS    = "sts.amazonaws.com"
 )
 
-func (c *awsClient) CreateOpenIDConnectProvider(providerURL string, thumbprint string) (string, error) {
+func (c *awsClient) CreateOpenIDConnectProvider(providerURL string, thumbprint string, clusterID string) (
+	string, error) {
 	output, err := c.iamClient.CreateOpenIDConnectProvider(&iam.CreateOpenIDConnectProviderInput{
 		ClientIDList: []*string{
 			aws.String(OIDCClientIDOpenShift),
 			aws.String(OIDCClientIDSTSAWS),
 		},
 		ThumbprintList: []*string{aws.String(thumbprint)},
 		Url:            aws.String(providerURL),
+		Tags: []*iam.Tag{
+			{
+				Key:   aws.String(tags.ClusterID),
+				Value: aws.String(clusterID),
+			},
+		},
```

Comment:
You need to ensure that the manual mode of `create oidc-provider` also contains these tags